lib/screen/home/home_page.dart

```diff
@@ -608,11 +608,15 @@ class HomePageState extends State<HomePage>
         .toList();
     final isRefresh =
         !listEquals(activeAddresses, NftCollectionBloc.activeAddress);
+    log.info("activeAddresses: $activeAddresses");
+    log.info(
+        "NftCollectionBloc.activeAddress: ${NftCollectionBloc.activeAddress}");
     if (isRefresh) {
       final listDifferents = activeAddresses
           .where(
               (element) => !NftCollectionBloc.activeAddress.contains(element))
           .toList();
+      log.info("listDifferents: ${listDifferents.toString()}");
```

Comment:
you should add the prefix `[HomePage]` to easier trace which class/file the log comes from.